server/terraform/README.md

```diff
@@ -118,8 +153,10 @@ This provides the config files that the Android and iOS apps need to communicate
 1. App nickname: "WHO COVID-19"
 1. Skip "Debug signing cert"... might be needed for staging apks
 1. "Register app"
-1. "Download google-services.json" and move to `<repo>/client/android/app/`
-1. TODO: need mechanism to switch between Firebase instances
+1. "Download google-services.json"
+1. Append name based on project, e.g. "google-services-staging.json"
+1. Add to repo: `<repo>/client/android/app/`
+1. TODO: need mechanism to switch between Firebase instances, default is staging server
```

Comment:
Yes, I noticed some paths being hard-coded into the app. Well, with an 'if'. Unfortunately I don't know enough Java to say how to embed this at build time or similar.

Reply:
This is a Firebase config issue in the client apps. There's probably some way to switch them but it hasn't be a priority to figure out.